### ISSUE TITLE
My server keeps crashing

### ISSUE THREAD

**Issue by ashtli10** [knokko/custom-items-gradle#233]
i have paper i tried spigot too i dont have a knokko core heres my console

```
WARN14:44:57
[CustomItems]: Task #43 for CustomItems v12.0 generated an exception
Console14:44:57
java.lang.NullPointerException: Cannot invoke "nl.knokko.customitems.plugin.data.PluginData.getCurrentTick()" because the return value of "nl.knokko.customitems.plugin.CustomItemsPlugin.getData()" is null
at nl.knokko.customitems.plugin.CustomItemsPlugin.lambda$onEnable$0(CustomItemsPlugin.java:127) ~[CustomItems.jar:?]
at nl.knokko.customitems.plugin.tasks.MobWands.cleanUp(MobWands.java:75) ~[CustomItems.jar:?]
at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.19.4.jar:git-Paper-524]
at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[paper-1.19.4.jar:git-Paper-524]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1483) ~[paper-1.19.4.jar:git-Paper-524]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:447) ~[paper-1.19.4.jar:git-Paper-524]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1397) ~[paper-1.19.4.jar:git-Paper-524]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1174) ~[paper-1.19.4.jar:git-Paper-524]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-524]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

and it spams

**Comment by knokko** [comment]
You need to export your item set with Editor 12.0 (or later) and follow the instructions. Once that's done, the errors should stop.

**Comment by knokko** [comment]
the error spam should be fixed in 12.1
